src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Tooltip/DefaultVSLSPTagHelperTooltipFactory.cs

```diff
@@ -11,6 +11,18 @@ namespace Microsoft.AspNetCore.Razor.LanguageServer.Tooltip
 {
     internal class DefaultVSLSPTagHelperTooltipFactory : VSLSPTagHelperTooltipFactory
     {
+        private static readonly Guid ImageCatalogGuid = new("{ae27a6b0-e345-4288-96df-5eaf394ee369}");
+
+        // Internal for testing
+        internal static readonly VSImageElement ClassGlyph = new(
+            new VSImageId(ImageCatalogGuid, 463), // KnownImageIds.Type = 463
```

Comment:
I assume they NEVER change these ID's?

Reply:
Nope, these are constants at the VS layer